### ISSUE TITLE
[AD] SCORPIO-Interface: Allow for output on dynamics grid

### ISSUE THREAD

**Issue by AaronDonahue** [E3SM-Project/scream#791]
This would be a nice feature to not lose sight of.  The ability to write fields used by dynamics on the native dynamics mesh. 

This is low priority because it is not essential for any release and may require a fair bit of thought on how to make sure that the written field is on the dynamics grid.

Most likely there will be a need to remap from physics->dynamics, or store a local copy of the field while it is on the dynamics grid that will be used by I/O later.  Or call the output from dynamics.

In short - there would need to be some design choices before this task could be implemented.

**Comment by bartgol** [comment]
Done in #1168 .
